.circleci/config.yml

```diff
@@ -546,6 +572,13 @@ workflows:
           <<: *release-tags-and-branches
   deploy:
     jobs:
+      - hold:
+          type: approval
```

Comment:
One question, if there are multiple commits in the release branch, will it launch a new job for each one and cancel the previous one?